samples/cpp/tutorial_code/ImgProc/Pyramids.cpp

```diff
@@ -28,20 +28,22 @@ int main( void )
   printf( " * [d] -> Zoom out \n" );
   printf( " * [ESC] -> Close program \n \n" );
 
-  /// Test image - Make sure it s divisible by 2^{n}
-  src = imread( "../data/chicky_512.png" );
+  //![load]
+  src = imread( "../data/chicky_512.png" ); // Test image - Make sure it s divisible by 2^{n}
```

Comment:
Why should it be divisible by 2^{n}, in case it is not, pixels at the border will just be lost right?